backend/src/api/interviews/interviews.api.ts

```diff
@@ -98,19 +100,43 @@ const initInterviewsApi: FastifyPluginAsync<Options> = async (
       body: interviewUpdateValidationSchema,
       params: interviewUpdateParamsValidationSchema,
     },
+    preHandler: checkHasPermissions('oneOf', PermissionKey.MANAGE_INTERVIEWS),
+    async handler(
+      req: FastifyRequest<{
+        Body: InterviewsUpdateRequestDto;
+        Params: InterviewsUpdateRequestParamsDto;
+      }>,
+      rep,
+    ) {
+      const interview = await interviewService.update({
+        id: req.params.id,
+        interviewUpdateInfoRequestDto: req.body,
+      });
+
+      return rep.status(HttpCode.OK).send(interview);
+    },
+  });
+
+  fastify.route({
+    method: HttpMethod.PUT,
+    url: InterviewsApiPath.$ID_UPDATE_WITHOUT_INTERVIEWER,
+    schema: {
+      body: interviewUpdateWithoutInterviewerValidationSchema,
+      params: interviewUpdateParamsValidationSchema,
+    },
     preHandler: checkHasPermissions(
       'oneOf',
-      PermissionKey.MANAGE_INTERVIEWS,
       PermissionKey.MANAGE_INTERVIEW,
+      PermissionKey.MANAGE_INTERVIEWS,
     ),
     async handler(
       req: FastifyRequest<{
-        Body: InterviewsUpdateRequestDto;
+        Body: InterviewsUpdateWithoutInterviewerRequestDto;
         Params: InterviewsUpdateRequestParamsDto;
       }>,
       rep,
     ) {
-      const interview = await interviewService.update({
+      const interview = await interviewService.updateWithoutInterviewer({
```

Comment:
To clarify, what if a user with `MANAGE_INTERVIEW` permission sends a request through Postman and includes ineterviewerId to body? I think the interviewer will change, have you checked that?